api/v1/db/oracledb/games-dao.js

```diff
@@ -0,0 +1,48 @@
+const appRoot = require('app-root-path');
+
+const { serializeGames } = require('../../serializers/games-serializer');
+
+const conn = appRoot.require('api/v1/db/oracledb/connection');
+
+/**
+ * @summary Return a list of games
+ * @function
+ * @returns {Promise<Object[]>} Promise object represents a list of games
+ */
+const getGames = async (queries) => {
+  // parse passed in parameters and construct query
+  const sqlParams = {
+    scoreMin: queries.scoreMin,
+    scoreMax: queries.scoreMax,
+  };
+  if (queries.name) {
+    sqlParams.name = queries.name;
+  }
+  if (queries.developerId) {
+    sqlParams.developerId = queries.developerId;
+  }
```

Comment:
The logic of these two if statements are actually the same. In order to prevent adding more repeating logic if we have more parameters supported in the future, can you figure a way to combine them together?

Reply:
I changed it to pull the list of parameters from the openapi and iterate through that.

Reply:
Oh I just saw your comment here. Why do we wanna get parameters from the openapi? Shouldn't they be retrieved from the queries?

Reply:
I wasn't clear enough. I grab a list of all parameter names specified in the openapi. I then iterate through that list and check if those parameters exist in `queries` and add them to `sqlParams` if they do. This way if a parameter is added to the openapi the JS code doesn't have to be changed.